content/advent-2016/cloudforest.md

```diff
@@ -0,0 +1,254 @@
+
+# Feature importance
+
+One of the most interesting aspects of building classification models with CloudForest is finding out which features have the biggest impact on your model. Not every feature impacts the model in the same way, some are highly correlated with what you're trying to predict and some not so much. Knowing which features are important can help you iterate and build better models.
+
+When you're running `growforest`, if you specify the flag `-importance=filename.tsv`, the `growforest` command will create a second output file along with the `.sf` file. This new file is a tsv file that lists every feature along with several metrics.
+
+Some of these metrics deal with how many trees chose to use a feature, others show how a feature impacted the accuracy of the whole model. The full list of metrics can be found [here](https://github.com/ryanbressler/CloudForest#importance).
+
+# Extra options for growforest
+
+To support all of the different ways CloudForest can build classification models, there are dozens of flags. The list of flags for all the options can feel daunting when you're not familiar with some of the terms, much like this:
+
+![CloudForest options](/postimages/advent-2016/cloudforest/controlpanel.png)
+
+However, some of the flags are of particular importance to any kind of model:
+
+- `nTrees`: the number of trees. By default, if you don't specify anything for this option, the forest will be built with `-nTrees=100`. A hundred trees if often a good enough number, but there is no one number of trees that will be appropriate for every situation.
+- `progress`: report the tree number and the running error. With `-progress`, it is possible to follow the construction of the forest. The first trees usually have high errors but the model quickly adjusts itself towards smaller and smaller errors as more trees are built. `-progress` is a good companion to `-nTrees`, it can be used to find out empirically how many trees are a good fit for your model. If the forest construction converges early, there is little benefit to increasing the number of trees.
+- `mTry`: the number of candidate features to be selected with each tree. Not every tree in a forest is build the same. Each classification tree is built with a subset of the features. `-mTry` determines how many features are available for selection during the construction of each tree. Lower values yield trees with less correlation and higher values allow more opportunities for important features to be selected. If not specified, `-mTry` is the square root of the number of features. This value is common across many implementations of Random Forests, and it's one of the parameters that has the highest impact on the accuracy of the final model. You may want to experiment with bigger values for `-mTry`. We've had success with much higher ratios, such as .5 (50% of the features available for each tree), but your mileage may vary.
```

Comment:
Should `is build` be `is built`, but then it seems a little repetitive with the sentence immediately after.